web/unit_tests/test.sh

```diff
@@ -124,9 +124,7 @@ BASE_PATH=`dirname $BASH_SOURCE`
 cd $BASE_PATH/../source
 
 # Compile our testing dependencies; make sure the script fails if compilation fails!
-./build_dev_resources.sh || fail "Dev resource compilation failed."
-cd ../tools/recorder
-./build.sh || fail "KMW recorder-module compilation failed."
+../tools/build.sh || fail "Dev resource compilation failed."
```

Comment:
Presumably you'll turn this into a dependency at some point when you update this script?

Reply:
That's the plan for the next round, yes.